src/app/(BottomTabNavigation)/Cases/QRCodeScanner/AddCase/index.tsx

```diff
@@ -0,0 +1,59 @@
+import { useLocalSearchParams, useRouter } from 'expo-router';
+import { useEffect, useState } from 'react';
+import { Image, Text, TouchableOpacity, View } from 'react-native';
+
+import styles from './styles';
+import {
+  containsDuplicateCase,
+  getImageUrl,
+  uploadCase,
+} from '../../../../../supabase/queries/cases';
+import { CaseUid } from '../../../../../types/types';
+
+function AddCase() {
+  const { id, title, summary } = useLocalSearchParams() as {
+    id: CaseUid;
+    title: string;
+    summary: string;
+  };
+  const router = useRouter();
+  const [loading, setLoading] = useState<boolean>(true);
+  const [image, setImage] = useState<string>();
+  console.log(loading);
+
+  useEffect(() => {
+    const waitForImage = async () => {
+      const bucketImage = await getImageUrl(id);
+      setImage(bucketImage);
+      setLoading(false);
+    };
+    waitForImage();
+  }, []);
+
+  const addToCases = async () => {
+    uploadCase(id);
+    router.push('/Cases');
+  };
+  if (loading) {
+    return (
+      <View style={styles.loading}>
+        <Text>LOADING... </Text>
+      </View>
+    );
+  }
+  return (
+    <View style={styles.container}>
```

Comment:
Let's make this scrollable! Summaries are likely a lot longer than the ones we put on Supabase.